keel-sqs/src/main/kotlin/com/netflix/spinnaker/config/SqsProperties.kt

```diff
@@ -0,0 +1,8 @@
+package com.netflix.spinnaker.config
+
+import org.springframework.boot.context.properties.ConfigurationProperties
+
+@ConfigurationProperties("sqs")
+class SqsProperties {
+  var queueARN: String = ""
```

Comment:
Do we also need to specify the topic ARN?

Reply:
Or, are you just creating a queue and pushing messages and reading from the queue?